content/applications/inventory_and_mrp/inventory/management/misc/cluster_picking.rst

```diff
@@ -0,0 +1,230 @@
+places the products directly in the package related to the customer order.
+
+This method is most efficient for medium-sized companies with high order volumes relatively few
+unique products, since the method eliminates the need for sorting products into packages for
+customers after picking. Disadvantages of cluster picking are that urgent orders cannot be
+prioritized and optimized batches must be manually created beforehand, so the picking process can
+lead to bottlenecks.
+
+.. _inventory/misc/cluster_picking/example:
+
+.. example::
+   #. |SO| 1 calls for one apple and orange
+   #. |SO| 2 requires one apple and banana
+   #. |SO| 3 needs one apple, orange, and banana
+
+   Apples are stored in Shelf A, oranges in Shelf B, and bananas in Shelf C.
+
+   To pick products for three orders at once, the cart is loaded with three empty packages.
+
+   Starting at Shelf A, the the picker places apples into each package. Next, the picker navigates
+   to Shelf B and places oranges in the packages designated for |SO| 1 and |SO| 3. Finally, the
+   picker pushes the cart to Shelf C and loads packages for |SO| 2 and |SO| 3 with a banana, each.
+
+   With all three |SOS| fulfilled, the picker pushes the cart to the output location, where
+   the packages are sealed and prepared for shipment.
+
+   .. image:: cluster_picking/cluster-example.png
+      :align: center
+      :alt: Show example of fulfilling sales orders 2 and 3 at once.
+
+Configuration
+=============
+
+To enable cluster picking, begin by navigating to :menuselection:`Inventory app --> Configuration
+--> Settings`. Under the :guilabel:`Operations` heading, activate the :guilabel:`Packages` and
+:guilabel:`Batch Transfers` options.
+
+.. image:: cluster_picking/configs.png
+   :align: center
+   :alt: Activate *Packages* and *Batch Transfers* features in the settings.
+
+Since batch picking is used to optimize the *pick* operation in Odoo, the :guilabel:`Storage
+Locations` and :guilabel:`Multi-Step Routes` options under the :guilabel:`Warehouse` heading must
+also be checked on this settings page. When finished, click :guilabel:`Save`.
+
+.. image:: cluster_picking/locations-routes-checkbox.png
+   :align: center
+   :alt: Enable *Storage Locations* and *Multi-Step Routes* Inventory > Configuration > Settings.
+
+.. _inventory/misc/create-package:
+
+Packages setup
+--------------
+
+After the :guilabel:`Packages` feature is enabled, navigate to :menuselection:`Inventory app -->
+Products --> Packages`, and click the :guilabel:`New` button to create a new package.
+
+On the new package form, the :guilabel:`Package Reference` is pre-filled with the next available
+`PACK` number in the system. :guilabel:`Pack Date` is automatically set to the creation date of the
+form.
+
+For the :guilabel:`Package Use` field, the :guilabel:`Disposable Box` radio option should be
+selected if the package is a shipping container. Alternatively, the :guilabel:`Reusable Box` option
```

Comment:
"Shipping container" makes me think of a big metal container like you'd see on a cargo ship. Again, a nitpick, but it might be worth addressing. There are multiple instances of this phrase throughout the doc.